### PR TITLE
Add `osu!monthly October 2022` news post

### PR DESCRIPTION
**Pull request by 0x84f**
## Self-check

- [x] Re-dated the file before going live
- [x] Added in the quiz question

### CODE REVIEW

**Comment by RockRoller01** [comment]
EGTS banner has @2x and @1x, but the other two community banners do not. Those two probably should have a @1x version added.

The skinning contest was mentioned in the september issue under "things to look out for in october", so the fact that it concluded should maybe be included under "community recap" (with a short recap ofc). Same story with the triangles beatmap contest

**Comment by 0x84f** [comment]
> EGTS banner has @2x and @1x, but the other two community banners do not. Those two probably should have a @1x version added.
> 
> The skinning contest was mentioned in the september issue under "things to look out for in october", so the fact that it concluded should maybe be included under "community recap" (with a short recap ofc). Same story with the triangles beatmap contest

We only have the @2x and @1x for the banners where we could get them made by the designers, unfortunately I didn't have time to reach out to everyone to ask for amended banners with @2x.

Skinning contests and the likes have their own news posts therefore we didn't really feel like it would fit to mention them multiple times, considering that people who look at osu!monthly most likely keep up with everything else being posted on the website.

File: news/2022-11-01-osumonthly-4.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Welcome to the 4th edition of osu!monthly, now with a new halloween themed banne
 While we are all currently focused on the osu! World Cup, we feel like it'd be good to take a look back at the month of October and see what else has been happening around the community. That being said, the world cup will be covered in great detail next month.
 
 If you want to contribute feel free to reach out! You can find us over at the [osu!dev Discord server](https://discord.com/invite/ppy), or you can reach out to 0x84f#0001 on Discord.
-Enjoy!
+If you are interested in checking out all of the previous editions feel free to look at our [forum discussion thread](https://osu.ppy.sh/community/forums/topics/1614314?n=1)!
```
Comment (wowcake) [inline news/2022-11-01-osumonthly-4.md:16]:
```suggestion
If you are interested in checking out all of the previous editions feel free to look at our [forum thread](https://osu.ppy.sh/community/forums/topics/1614314?n=1)!
```

Comment (wowcake) [reply news/2022-11-01-osumonthly-4.md:16]: no longer a discussion thread and more of a "wiki" / "hub" thread instead 
(all the discussions are happening in the #osu-news channel)

**Review by wowcake — CHANGES_REQUESTED** [review]

**Comment by Ephemeralis** [comment]
Applied a fairly beefy set of changes to streamline some wording and other miscellaneous stuff.

For the future, when sharing plays for all game modes, the established convention for any kind of game mode sorting is osu! -> osu!taiko -> osu!catch -> osu!mania, as this mirrors their integer assignments in code and also their release dates, I believe.

Game modes should always have osu! prefixed in front of them too (taiko becomes osu!taiko, catch becomes osu!catch, mania becomes osu!mania, etc). lazer should always be osu!(lazer) as well (don't look at me for this one, I am but a humble cog in the machine).

Generally, using past tense for nearly everything in these articles is appropriate unless they're referencing an ongoing event. The osu!monthly comes out at the end of the month, after all.

**Comment by 0x84f** [comment]
Thanks, I really didn't have too much time to check everything due to family matters that needed attending so missed a bunch of little things. One thing that remains to be updated is the medals mention as the community had uncovered more medals (as I am sure you are aware).

**Pull request merged**